### PR TITLE
Migrate to Profile Dashboard from dev

### PR DESCRIPTION
**Pull request by mjoyce91**
Migrates to the old profile dashboard.

Relies on https://github.com/18F/State-TalentMAP/pull/1837

### CODE REVIEW

File: src/Components/ProfileDashboard/BidList/BidListHeader/BidListHeader.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import FontAwesome from 'react-fontawesome';
 // Due date is static for now
 const BidListHeader = () => (
   <div className="usa-grid-full bid-list-header section-padded-inner-container-narrow">
-    <div className="usa-width-two-thirds">
+    <div className="usa-width-two-thirds bid-list-header-text">
       <FontAwesome name="clock-o" /> All bids are due 7.15.18
```
Comment (Icemanbeta) [inline src/Components/ProfileDashboard/BidList/BidListHeader/BidListHeader.jsx]: I know this wasn't done in this PR but should this really be hard coded?

File: src/Components/ProfileDashboard/ProfileDashboard.jsx [hunk 2]
```diff
@@ -1,48 +1,58 @@
...
 
 const ProfileDashboard = ({
-  userProfile,
-  isLoading,
-  favoritePositions,
-  favoritePositionsIsLoading,
+  userProfile, isLoading, assignment, assignmentIsLoading, notifications,
+  notificationsIsLoading, bidList, bidListIsLoading,
 }) => (
   <div className="usa-grid-full user-dashboard user-dashboard-main profile-content-inner-container">
```
Comment (Icemanbeta) [inline src/Components/ProfileDashboard/ProfileDashboard.jsx]: Use `Row`, `Column`! It'll make me feel good lol

File: src/sass/_avatar.scss [hunk 3]
```diff
@@ -9,3 +9,10 @@
   text-align: center;
   width: $avatar-size;
 }
```
Comment (Icemanbeta) [inline src/sass/_avatar.scss]: Why don't we set a variable to handle small sizing? And a simple variable expression how you came to `19px` besides visually. 50% - 1px seems reasonable to me on this mainly cause odd base numerics.

Comment (Icemanbeta) [inline src/sass/_avatar.scss]: Why don't we set a variable to handle small sizing? And a simple variable expression how you came to `19px` besides visually. 50% - 1px seems reasonable to me on this mainly cause odd base numerics.

File: src/sass/_avatar.scss [hunk 4]
```diff
@@ -9,3 +9,10 @@
...
+
+.tm-avatar--small {
+  font-size: 19px;
+  height: 40px;
+  line-height: 40px;
+  width: 40px;
+}
```
Comment (Icemanbeta) [inline src/sass/_avatar.scss]: Why don't we set a variable to handle small sizing? And a simple variable expression how you came to 19px besides visually. 50% - 1px seems reasonable to me on this mainly cause odd base numerics.

File: src/Components/ProfileDashboard/BidList/BidListHeader/BidListHeader.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/BidList/BidListHeader/BidListHeader.jsx]: Just static text for demo'ing purposes for now

File: src/sass/_avatar.scss (hunk 4 above)
Comment (mjoyce91) [reply src/sass/_avatar.scss]: I was thinking about that, but it seems like our sizing is so specific based on the context that we're using the Avatar in. I don't think there's currently any repetition of size.

File: src/Components/ProfileDashboard/ProfileDashboard.jsx (hunk 2 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/ProfileDashboard.jsx]: I used your approach of using Rows, Columns and MediaQueryWrapper in `SavedSearchesListResultsCard` and it looks much better! Thanks for the suggestion.

Comment (mjoyce91) [reply src/Components/ProfileDashboard/ProfileDashboard.jsx]: https://github.com/18F/State-TalentMAP/pull/1842/commits/5f362405038473959aadf2ccfe861a6aa87a7ffc

**Comment by mjoyce91** [comment]
@burgwyn I think we're good to merge

**Pull request merged**